guide/designing-products/accessibility.md

```diff
@@ -0,0 +1,149 @@
+
+Full or partial blindness, color blindness and other visual disabilities. Three million people in the US alone have vision impairment and one million people are blind ([source](https://www.afb.org/research-and-initiatives/statistics)). See how a [blind person uses an iPhone](https://youtu.be/fjdcKioHb5w).
+
+#### Physical
+
+A physical disability can effect a person temporarily or permanently. On a practical level this affects a their ability to use a keyboard or mouse. Tapping a screen or holding a phone still to scan a QR code could pose as  a challenge. Seventy-five million people across the world need a wheel chair on a daily basis ([source](https://www.inclusivecitymaker.com/disabled-people-in-the-world-in-2021-facts-and-figures/)). Listen to a [quadriplegic talking about accessibility](https://www.youtube.com/watch?v=eMFrcJBX0pA).
+
+#### Intellectual
+
+Learning difficulties refer to a lower ability to understand new or complex information. Someone with intellectual difficulties also struggles with learning and applying new skills and can also have difficulty reading or understanding things. Over two hundred million people have some form of intellectual disability ([source](https://www.inclusivecitymaker.com/disabled-people-in-the-world-in-2021-facts-and-figures/)).
+
+#### Hearing
+
+Hearing difficulty such as being partly deaf, this can affect one or both ears. Any hearing loss below 20 decibels is considered a hearing loss. 400 million people have some form of deafness or hearing loss.([source](https://www.inclusivecitymaker.com/disabled-people-in-the-world-in-2021-facts-and-figures/))
+
+#### Literacy
+
+Illiteracy is the inability of a person to read/write or understand a simple statement that is related to his/her everyday life. There are over 700 million illiterate adults around the world and more than half of them are women. ([source](http://uis.unesco.org/en/topic/literacy))
+
+## Tips and examples
+
+Improving accessibility may seem like an overwhelming task. The most important thing is to get started, so let's take a look at some basics. Correctly labeling interactive elements is one of the essential practices in accessibility. It allows screen readers to understand and convey the information to impaired users in various ways. The following example illustrates some good and bad practices.
+
+{% include image-gallery.html pages = page.exampleImages %}
+
+It is impossible to know what a button does when it is only labelled as "button", and not with its function. Testing has shown that QR codes are often not selectable at all, which makes them invisible to users reliant on screen readers. A good solution is a label such as "QR code to copy the receive address".
+
+Also important is that elements are correctly ordered. Otherwise users reliant on the keyboard, or other linear navigation techniques, will not be able to easily understand what is on the screen.
+
+{% include image-gallery.html pages = page.exampleImagesTwo %}
```

Comment:
There's a ton of tiny detail in these images. This would be the perfect place to use the [modal images feature](https://bitcoin.design/guide/contribute/formatting/#image-viewable-in-larger-modal).

Reply:
Modals are added.